src/app/Routes/DistributionRoutes.tsx

```diff
@@ -16,12 +14,12 @@ const DistributionRoutes: any = () => {
 
   return (
     <Layout>
-      <AppBar excludeNetworks={[HOLESKY_NETWORK_ID]} />
```

Comment:
make sure we don't have holesky button on faucet app

Reply:
"Now we choose the networks for each app in the Wagmi config"